common/locales/en-US/main.ftl

```diff
@@ -35,6 +35,15 @@ uplink = Uplink
     .check-for-updates = Check for updates
     .download-update = Download Update
 
+updates = Updates
+    .title = Uplink has an update available. Follow these steps to download and install the latest update.
+    .instruction1 = 1. Download the update from our release page.
+    .instruction2 = 2. Close Uplink.
+    .instruction3 = 3. Run the installer. The older version of uplink will be overwritten, leaving your account intact.
+    .instruction4 = 4. Open Uplink and start chatting again!
+    .button-label = Pick Download Folder
+    .download-label = Download Update
+
```

Comment:
do we have/are we gonna add force update?  🔨

Reply:
not sure
for now all manual